test/functional/mining_prioritisetransaction.py

```diff
@@ -116,7 +116,7 @@ def run_test(self):
         tx_id = self.nodes[0].decoderawtransaction(tx_hex)["txid"]
 
         # This will raise an exception due to min relay fee not being met
-        assert_raises_rpc_error(-26, "min relay fee not met, 0 < 134 (code 66)", self.nodes[0].sendrawtransaction, tx_hex)
+        assert_raises_rpc_error(-26, "min relay fee not met, 0 < 191 (code 66)", self.nodes[0].sendrawtransaction, tx_hex)
```

Comment:
This should say `... 192 ...` actually :) let's just drop 32b94522ad7616a23861c59d7ae84bb0a9336510 and backport https://github.com/bitcoin/bitcoin/pull/13082 instead.

Reply:
This ^^^ is still important to fix tests...

Reply:
@UdjinM6 Just curious since tests passed with `191`, why do you think about `192`?

Reply:
See raw logs for https://gitlab.com/dashpay/dash/-/jobs/846503979

```
[1mmining_prioritisetransaction.py[0m failed, Duration: 3 s

[1mstdout:
[0m2020-11-12 20:09:13.607000 TestFramework (INFO): Initializing test directory /builds/dashpay/dash/build-ci/dashcore-linux32/testdatadirs/dash_test_runner_20201112_200229/mining_prioritisetransaction_419
2020-11-12 20:09:16.034000 TestFramework (INFO): Assert that prioritised transaction was mined
2020-11-12 20:09:16.311000 TestFramework (INFO): Assert that de-prioritised transaction is still in mempool
2020-11-12 20:09:16.342000 TestFramework (ERROR): Assertion failed
Traceback (most recent call last):
  File "/builds/dashpay/dash/build-ci/dashcore-linux32/test/functional/test_framework/util.py", line 124, in try_rpc
    fun(*args, **kwds)
  File "/builds/dashpay/dash/build-ci/dashcore-linux32/test/functional/test_framework/coverage.py", line 47, in __call__
    return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)
  File "/builds/dashpay/dash/build-ci/dashcore-linux32/test/functional/test_framework/authproxy.py", line 138, in __call__
    raise JSONRPCException(response['error'])
test_framework.authproxy.JSONRPCException: min relay fee not met, 0 < 192 (code 66) (-26)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/builds/dashpay/dash/build-ci/dashcore-linux32/test/functional/test_framework/test_framework.py", line 156, in main
    self.run_test()
  File "/builds/dashpay/dash/build-ci/dashcore-linux32/test/functional/mining_prioritisetransaction.py", line 119, in run_test
    assert_raises_rpc_error(-26, "min relay fee not met, 0 < 191 (code 66)", self.nodes[0].sendrawtransaction, tx_hex)
  File "/builds/dashpay/dash/build-ci/dashcore-linux32/test/functional/test_framework/util.py", line 116, in assert_raises_rpc_error
    assert try_rpc(code, message, fun, *args, **kwds), "No exception raised"
  File "/builds/dashpay/dash/build-ci/dashcore-linux32/test/functional/test_framework/util.py", line 130, in try_rpc
    raise AssertionError("Expected substring not found:" + e.error['message'])
AssertionError: Expected substring not found:min relay fee not met, 0 < 192 (code 66)
2020-11-12 20:09:16.343000 TestFramework (INFO): Stopping nodes
2020-11-12 20:09:16.946000 TestFramework (WARNING): Not cleaning up dir /builds/dashpay/dash/build-ci/dashcore-linux32/testdatadirs/dash_test_runner_20201112_200229/mining_prioritisetransaction_419
2020-11-12 20:09:16.946000 TestFramework (ERROR): Test failed. Test logging available at /builds/dashpay/dash/build-ci/dashcore-linux32/testdatadirs/dash_test_runner_20201112_200229/mining_prioritisetransaction_419/test_framework.log
2020-11-12 20:09:16.947000 TestFramework (ERROR): Hint: Call /builds/dashpay/dash/build-ci/dashcore-linux32/test/functional/combine_logs.py '/builds/dashpay/dash/build-ci/dashcore-linux32/testdatadirs/dash_test_runner_20201112_200229/mining_prioritisetransaction_419' to consolidate all logs
```

Reply:
it can actually be 191 or 192 based on the sig size (?) I guess, so the right thing is to either check it based on the tx size or to just drop this part altogether.